cpu/kinetis/include/periph_cpu.h

```diff
@@ -165,18 +186,21 @@ typedef enum {
     GPIO_PCR_PD    = (PORT_PCR_PE_MASK),    /**< enable pull-down */
     GPIO_PCR_PU    = (PORT_PCR_PE_MASK | PORT_PCR_PS_MASK)  /**< enable PU */
 } gpio_pcr_t;
+#endif //KINETIS_HAVE_PCR
```

Comment:
please change to `/* ... */` here and below, I counted 8 "wrong" comment over all files